src/services/users/index.js

```diff
@@ -123,10 +129,39 @@ const inviteUser = async (email) => {
 	return signupInfo
 }
 
+const getNewTokens = async (email, oldRefreshToken) => {
+	const user = await getUserByEmail(email)
+
+	const isrefreshTokenSame = await verifyRefreshToken (
+		oldRefreshToken, 
+		user.refreshToken)
+	if (!isrefreshTokenSame) {
+		throw new AppError(unauthorized , 401)
```

Comment:
kindly go with the custom error style we did for the other services

 the service should ideally not be concerned with status codes

Reply:
yes please, thank you